src/TableHeaderCell.tsx

```diff
@@ -9,7 +9,8 @@ export interface TableHeaderCellProps {
 /**
  * HeaderCell component for BaseTable
  */
-const TableHeaderCell: React.FunctionComponent<TableHeaderCellProps> = ({ className, column, columnIndex }) => 
+export type TTableHeaderCell = React.FunctionComponent<ITableHeaderCellProps>;
```

Comment:
Lets avoid this intermidaite thing. Here it is so problematic that you are just exporting the type and not the actual implementation, tricking typescript to compile but when you import from this file you will get undefined

Reply:
But that type is also used in `BaseTable.tsx, line:1242`. Don't you think its better to define once and reuse rather than having several instances of `React.FunctionComponent<ITableHeaderCellProps>`, etc ?